source/create_utplsql_owner.sql

```diff
@@ -27,9 +27,10 @@ define ut3_tablespace = &3
 
 create user &ut3_user identified by &ut3_password default tablespace &ut3_tablespace quota unlimited on &ut3_tablespace;
 
-grant create session, create sequence, create procedure, create type, create table, create view, create synonym to &ut3_user;
+grant create session, create sequence, create procedure, create type, create table, create view, create synonym, select any dictionary to &ut3_user;
```

Comment:
Aren't we're trying to satisfy DBA's needs too much?
I fear that If we will try to satisfy everyone's needs we will not progress and keep on stumbling upon issues.
We would probably need to separate coverage reporting capability from the main framework and make it a separate add-on but I don't really like the idea.

**How should we separate this privilege in your opinion?**

We could change the privilege to be one of:
- `select catalog role`
- `select on dba_source` (preffered)
- `select on all_source` (not so good, but used currently)
- `select_any_dictionary` (not tested)